packages/web-config-server/src/apiV1/organisationUnit.js

```diff
@@ -12,12 +12,46 @@ const translateDescendantForFrontEnd = (descendant, childIdToParentId, entityIdT
   parent: entityIdToCode[childIdToParentId[descendant.id]],
 });
 
+async function fetchEntitiesWithProjectAccess(req, entities, userGroups) {
+  return Promise.all(
+    entities.map(async ({ id, name, code, bounds }) => ({
+      id,
+      name,
+      code,
+      bounds,
+      hasAccess: await Promise.all(userGroups.map(u => req.userHasAccess(code, u))),
+    })),
+  );
+}
+
 export default class extends RouteHandler {
   static PermissionsChecker = PermissionsChecker; // checks the user has access to requested entity
 
   async buildResponse() {
     const { includeCountryData } = this.query;
     const project = await this.fetchProject();
+
+    // If this is a project entity, get entitiesWithAccess and set them on the entity
+    if (this.entity.isProject()) {
+      const orgUnits = await this.entity.getNearestOrgUnitDescendants();
+
+      const accessByEntity = await Promise.all(
+        orgUnits.map(async orgUnit => {
+          const { id, name, code, bounds } = orgUnit;
+          return {
+            id,
+            name,
+            code,
+            bounds,
+            hasAccess: await this.checkUserHasEntityAccess(orgUnit),
+          };
```

Comment:
This is not a full entity, but it's also a lot more than we need (as far as I can tell). Could we go one way or the other? I'd opt for the minimal version, in which case the result would be more like `boundsOfEntitiesWithAccess`.